### DISCUSSION TITLE
BE/SpringBoot/Subsubcategory%202/post-05

### DISCUSSION THREAD

**Discussion by giscus** [byanko55/jekyll-theme-satellite#48] [Q&A]
# BE/SpringBoot/Subsubcategory%202/post-05

About the book Author: Miguel de Cervantes Original title: El ingenioso hidalgo don Quixote de la Mancha Country: Spain Genre: Novel Publication date: 1605 (Part One) 1615 (Part Two)

https://theuniongraphix.github.io/BE/SpringBoot/Subsubcategory%202/post-05.html

**Comment by Ju-Yeongmin** [comment]
좋은 게시물이네요!!
